slave/redhat-openstack/fabfile.py

```diff
@@ -8,6 +8,7 @@
 from pipes import quote as shellQuote
 from fabric.api import sudo, task, env, put, run, local
 from fabric.context_managers import shell_env
+from cuisine import file_update, text_ensure_line, mode_sudo, mode_remote
```

Comment:
I'm not sure about this new dependency.  Is it so much harder to do these things using fabric directly?